src/v/cluster/members_table.cc

```diff
@@ -94,7 +94,7 @@ std::error_code members_table::apply(model::offset o, add_node_cmd cmd) {
 }
 
 void members_table::set_initial_brokers(std::vector<model::broker> brokers) {
-    vassert(!_nodes.empty(), "can not initialize not empty members table");
+    vassert(_nodes.empty(), "can not initialize not empty members table");
```

Comment:
Hmm, how was it working before? :) Looks like this assert is firing in tests

Reply:
there is a bug here. I think it is all working because of the race condition in feature management